config/helm/appmesh-controller/test.yaml

```diff
@@ -0,0 +1,146 @@
+  envoyAdminAccessLogFile: /tmp/envoy_admin_access.log
+  resources:
+    # sidecar.resources.requests: Envoy CPU and memory requests
+    requests:
+      cpu: 10m
+      memory: 32Mi
+    # sidecar.resources/limits: Envoy CPU and memory limits
+    limits:
+      cpu: 100m
+      memory: 64Mi
+  lifecycleHooks:
+    # sidecar.lifecycleHooks: Envoy PreStop Hook Delay
+    preStopDelay: 20
+  probes:
+    # sidecar.probes: Envoy Readiness Probe
+    readinessProbeInitialDelay: 1
+    readinessProbePeriod: 10
+init:
+  image:
+    repository: 840364872350.dkr.ecr.us-west-2.amazonaws.com/aws-appmesh-proxy-route-manager
+    tag: v3-prod
+
+xray:
+  image:
+    repository: amazon/aws-xray-daemon
+    tag: latest
+
+nameOverride: ""
+fullnameOverride: ""
+
+resources:
+  limits:
+    cpu: 2000m
+    memory: 1Gi
+  requests:
+    cpu: 100m
+    memory: 200Mi
+
+nodeSelector: {
+    test: test
+}
+
+tolerations:
+  - key: "key1"
+    operator: "Equal"
+    value: "value1"
+    effect: "NoExecute"
+
+affinity: {
+    test: test
+}
+
+podAnnotations: {
+    test: test
+}
+
+podLabels: {
+    test: test
+}
+
+cloudMapCustomHealthCheck:
+  # cloudMapCustomHealthCheck.enabled: `true` if CustomHealthCheck needs to be enabled in CloudMap
+  enabled: true
+
+cloudMapDNS:
+  # cloudMapDNS.ttl if set will use this global ttl value
+  ttl: 300
+
+sds:
+  # sds.enabled: `true` if SDS based mTLS support needs to be enabled in envoy
+  enabled: true
+  #sds.udsPath: UDS Path of the SDS Provider. Default value is tied to SPIRE.
+  udsPath: /run/spire/sockets/agent.sock
+
+serviceAccount:
+  # serviceAccount.create: Whether to create a service account or not
+  create: true
+  # serviceAccount.name: The name of the service account to create or use
+  name: test
+  # serviceAccount.annotations: optional annotations to be applied to service account
+  annotations: {
+      test: test
+  }
+
+rbac:
+  # rbac.create: `true` if rbac resources should be created
+  create: true
+  # rbac.pspEnabled: `true` if PodSecurityPolicy resources should be created
+  pspEnabled: true
+
+log:
+  #log.level: info (default), debug
+  level: "info"
+
+tracing:
+  # tracing.enabled: `true` if Envoy should be configured tracing
+  enabled: true
+  # tracing.provider: can be x-ray, jaeger or datadog
+  provider: x-ray
+  # tracing.address: Jaeger or Datadog agent server address (ignored for X-Ray)
+  address: appmesh-jaeger.appmesh-system
+  # tracing.port: Jaeger or Datadog agent server port
+  port: 2000
+
+stats:
+  # stats.tagsEnabled: `true` if Envoy should include app-mesh tags
+  tagsEnabled: false
+  # stats.statsdEnabled: `true` if Envoy should publish stats to statsd endpoint @ 127.0.0.1:8125
+  statsdEnabled: false
+  #stats.statsdAddress: DogStatsD daemon address
+  statsdAddress: 127.0.0.1
+  #stats.statsdPort: DogStatsD daemon port
+  statsdPort: 8125
+
+# Enable cert-manager
+enableCertManager: false
+
+# Environment variables to set in appmesh-controller pod
+env: {}
```

Comment:
I think we should add Liveness probe setting here as well. In accordance to this PR: https://github.com/aws/aws-app-mesh-controller-for-k8s/pull/436/commits/0094ba69b6668a3c6a97af1a75211f2f702e92ed

Reply:
Yes, since the liveness probe PR https://github.com/aws/aws-app-mesh-controller-for-k8s/pull/436 is not merged, I haven't included. Let's add it once it is merged